### PR TITLE
Update SIMPLISAFE_EVENT results and automation usage.

### PR DESCRIPTION
**Pull request by tanderson1992**
The original documentation shows SIMPLISAFE_EVENT being an event_type and also serving an event_type as a subset. The current simplipy version has an event_type of SIMPLISAFE_EVENT, and the event_data returned is almost all prefaced with "last_". Since this document deals with v2 and v3, it's possible the original documentation displays what a v2 system will show. If that's the case, we should include both sets of event data with a header describing which goes with which Simplisafe version.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @tanderson1992, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16342).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/simplisafe.markdown [hunk 1]
```diff
@@ -105,30 +105,51 @@ following keys:
...
 * `camera_motion_detected`
 * `doorbell_detected`
 * `entry_detected`
 * `motion_detected`
 
+To build an automation using one of these, use `SIMPLISAFE_EVENT`
+as an event trigger, with `last_event_type` as the `event_data`.
+For example, the following will trigger when the doorbell rings:
+
+```
```
Comment (frenck) [inline source/_integrations/simplisafe.markdown]: Missing language identifier. e.g., ` ```yaml `

**Comment by frenck** [comment]
Make sure the builds pass 😄 

I've requested the codeowner of the simplisafe integration to take a look (@bachya).

**Comment by tanderson1992** [comment]
Thanks. This is literally the first time I've submitted a documentation change with Github. I was trying to knock it out before going to bed last night and figured I'd pick up again today with trying to figure out where it went wrong. I posted that I would submit this change to @bachya  on the HA forums: https://community.home-assistant.io/t/simplisafe-very-unreliable-as-of-late/255013/27?u=tanderson1992.

File: source/_integrations/simplisafe.markdown (hunk 1 above)
Comment (tanderson1992) [reply source/_integrations/simplisafe.markdown]: Thanks, added `yaml`

**Review by bachya — APPROVED** [review]
Content looks good!

**Review by frenck — APPROVED** [review]
Alright! Thanks, @tanderson1992 👍

**Pull request merged**